Source/DafnyLanguageServer.Test/Synchronization/IncludeFileTest.cs

```diff
@@ -69,7 +69,6 @@ public class IncludeFileTest : ClientBasedLanguageServerTest {
   [Fact]
   public async Task MethodWhosePostConditionFailsAndDependsOnIncludedFile() {
     var temp = (Path.GetTempFileName() + ".dfy").Replace("\\", "/");
-    Console.WriteLine("temp file is: " + temp);
```

Comment:
Just curious - was this causing issues or was it just a leftover debug artifact?

Reply:
The latter